lib/screens/login/login_screen.dart

```diff
@@ -117,6 +117,20 @@ class _LoginScreenState extends ConsumerState<LoginScreen>
       next.maybeWhen(
         data: (_) {
           // TODO: Navigate to the Home screen
+          showAlertDialog(
+            context: context,
+            title: 'Login Successfully',
+            message: 'You are now logged in!',
```

Comment:
This dialog is shown temporarily because we didn't implement the Home screen at the time.
Updated with https://github.com/nimblehq/flutter-ic-khanh-thieu/pull/57/commits/8343044d7ededb0bc84a3cae88187427344788f4 and POW for navigating to the Home screen 🙏